### PR TITLE
Fix typos in many files (Addresses #149) (COMMUNITY)

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description
This PR fixes many typos all over the project files. The typos were found on https://fossies.org/linux/test/cwa-app-ios-development-snap.tar.gz/codespell.html. 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9775

## Additional information
This PR is quite huge, it changes 85 files. I don't want to be the one with the PR which causes merge conflicts, therefore, please merge all PRs that have to be merged, I will resolve any merge conflicts which may happen on this PR as a consequence of other merges.

I hope this changes is welcome, I tackled most of the typos shown on [fossies.org](https://fossies.org/linux/test/cwa-app-ios-development-snap.tar.gz/codespell.html), despite some which were in the code itself (and not in comments, etc.) or the ones where I was unsure what the correct word is.

I verified on my simulator that the build still works. However, I can't say anything reg. the tests, as they always fail with the error "'HealthCertificateToolkit was not compiled for testing" on my environment.

---
Internal Tracking ID: [EXPOSUREAPP-9775](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9775)

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ struct Country: Equatable, Codable {
 		UIImage(named: "flag.\(id.lowercased())")
 	}
 
-	/// Initialize a country with a given `countryCode`. If no valid `countryCode` is given the initalizer returns `nil`.
+	/// Initialize a country with a given `countryCode`. If no valid `countryCode` is given the initializer	 returns `nil`.
```
Comment (heinezen) [inline src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift:24]: Accidental <kbd>Tab</kbd>?

**Review by heinezen — DISMISSED** [review]

File: src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift (hunk 1 above)
Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift:24]: @heinezen

Do you mean at the start or at the end of the line? Cause the tab in at the start was already there and I'm not seeing a tab added. 😅

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift:24]: Btw can you add the `Community` label here @heinezen?

Comment (heinezen) [reply src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift:24]: @Ein-Tim There is a tab right after "initializer": `initializer<TAB>`

You can kind of see it in the Github diff.

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/Models/Exposure/Country.swift]: @heinezen 

Ah, now I know what you mean! Fixed in [`a44dbda` (#3627)](https://github.com/corona-warn-app/cwa-app-ios/pull/3627/commits/a44dbda88f7760d97d58b695e0499227eb2f50cd).

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift [hunk 2]
```diff
@@ -129,7 +129,7 @@ enum AccessibilityIdentifiers {
 		}
 		
 		enum NotesInformation {
-			static let titel = "AppStrings.ContactDiary.NotesInformation.title"
+			static let title = "AppStrings.ContactDiary.NotesInformation.title"
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:132]: Have you already checked if the spelling change is not affecting where the identifier is used.

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift (hunk 2 above)
Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:132]: I will check @naveeddotio.

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift:130]: Mh, I have no way on an Emulator to check whether the title is still spoken when the text gets read out, sorry. 
Maybe you could check on a real device @naveeddotio?

I can also offer to revert the change on this file.

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift]: @naveeddotio I reverted the change to this file. Please review again.

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**